css/style.css

```diff
@@ -1,4 +1,144 @@
-h1 {
-  font-family: Arial, Helvetica, sans-serif;
-  color: rgb(50, 0, 200);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main {
+  background: url(../img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] **REQUIRED:** Your CSS looks good, but I offer you to follow [DRY](https://vanseodesign.com/css/dry-principles/) rule, for example, create selector like 

```

.d-flex {
display: flex;
align-items: center;
}

```

And whenever you need that like style for any element in your project, you can add this class to that element